Application/src/tracker/python/trex_learn_category.py

```
@@ -340,25 +353,42 @@ def perform_training(self):
+
+        y_pred, avg_loss = self._predict_labels_and_loss(X_test, Y_test, batch_size)
+
+        #TRex.log("# y_pred: "+str(y_pred.shape)+" y_pred:"+str(y_pred))
+        TRex.log("#ypred/class: "+str(np.unique(y_pred.cpu().numpy(), return_counts=True)))
+
+        report = classification_report(Y_test.cpu().numpy(), y_pred.cpu().numpy(), output_dict=True, zero_division=0)
 
         for key in report:
             TRex.log("report: "+str(key)+" "+str(report[key]))
         TRex.log(str(report))
         set_best_accuracy(float(report["accuracy"]))
```

Ruff (0.14.8)

Undefined name `set_best_accuracy`

(F821)